drape/gl_functions.cpp

```diff
@@ -255,8 +255,16 @@ void GLFunctions::Init(dp::ApiVersion apiVersion)
     glUnmapBufferFn = &::glUnmapBuffer;
 
 #elif defined(OMIM_OS_LINUX)
-
-    CHECK(false, ("OpenGLES2 is not yet supported"));
+    void *libhandle = dlopen("libGL.so.1", RTLD_NOW);
```

Comment:
Is it portable?

Reply:
At the moment I can't think of any typical scenario in which it wouldn't work. 
Perhaps some rare corner cases, in which there are multiple displays attached to different GPUs,
which can only do ES 2.0.
But ES 2.0 only hardware is rarer and rarer anyway, any such corner cases are unsupported anyway.
At the moment I don't consider it a high priority to cover corner cases in which this wouldn't work.
I think if the project wants to keep up with technological development at large,
then we should make this ES 2.0 support leap as soon as possible, and move on to ES 3.0 and beyond,
and see if anyone is even interested in some obscure, old setup.

Reply:
1. Does it make sense to support GLES and (non-ES) GL at the same time, for desktops and mobile linux?
2. Where only ES2 is used? How many devices won't be able to run OM if we drop it?